File: week1/q1.c
#include <stdio.h>

int main() {
    int n = 4;
    int arr[5] = {10, 20, 30, 40, 0};
    int ele = 50;
    
    printf("Array before insertion\n");
    for (int i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }

   
    arr[n] = ele;

    printf("\nArray after insertion\n");
    for (int i = 0; i <= n; i++) {
        printf("%d ", arr[i]);
    }

    return 0;
}

File: week1/q1.py
if __name__ == "__main__":
    arr = [10, 20, 30, 40]
    ele = 50
    print("Array before insertion")
    for i in range(len(arr)):
        print(arr[i], end=" ")

    # Insert element at the end
    arr.append(ele)

    print("\nArray after insertion")
    for i in range(len(arr)):
        print(arr[i], end=" ")